WDL/CLI.py

```diff
@@ -390,6 +412,10 @@ def cromwell(uri, inputs, json_only, empty, check_quant, rundir=None, path=None,
         "-i",
         input_json_filename,
     ]
+    
+    if config_setting:
+        cromwell_cmd.insert(3, config_setting)
```

Comment:
We can build up `cromwell_cmd` in stages to make this a little less brittle, e.g.

```
cromwell_cmd = ["java", "-DLOG_LEVEL=warn", "-DLOG_LEVEL=pretty"]
cromwell_cmd.extend([config_setting] if config_setting else [])
cromwell_cmd.extend(["-jar", jarpath, ...])
```